### DISCUSSION TITLE
Theme switcher

### DISCUSSION THREAD

**Discussion by nbouliol** [saadeghi/daisyui#1903] [Q&A] [answered]
Hello, 

Thanks for this awesome lib !

The theme switcher on the daisyUI website is really great, as it let the user preview the colors of the themes. 

Is it possible to get it as a component ?

**Comment by saadeghi** [comment, marked as answer]
It's using [theme-change](https://github.com/saadeghi/theme-change) script.  
And the HTML itself is just divs with `data-theme` for each theme so after adding `theme-change` you can copy the theme switcher's HTML and it works for your site as well.

(You should also need to set [`themes: true` config](https://daisyui.com/docs/config/) to enable all themes.)
